package.json

```diff
@@ -6,7 +6,12 @@
   "scripts": {
     "test": "npx hardhat --network localhost test",
     "lint": "eslint $(git ls-files '*.ts')",
-    "format": "prettier-eslint --list-different --write $(git ls-files '*.ts')"
+    "format": "prettier-eslint --list-different --write $(git ls-files '*.ts')",
```

Comment:
We should use this in CI/CD.

Reply:
https://github.com/0xSpaceShard/starknet-hardhat-example/pull/122/commits/81a239b8b7af39d0241991f48bf1e5d170bc7667